source/deploying-with-kubernetes/kubernetes-conf.rst

```diff
@@ -172,11 +172,15 @@ Deploy
 
     3.2. Apply all manifests using kustomize
 
-        By using the kustomization.yml we can now deploy the whole cluster in a single command.
+        We are using the overlay feature of Kustomize to create two variants: ``eks`` and ``local-env``, in this guide we're using ``eks``. (For a deployment on a local environment check the guide :ref:`Deployment on local environment <kubernetes_local_env>`)
```

Comment:
It could be a good idea to link the Kubernetes documentation related to Kustomize feature https://kubernetes.io/docs/tasks/manage-kubernetes-objects/kustomization/

Reply:
Fixed in 25aeae774